templates/cisco_ios_show_alert_counters.textfsm

```diff
@@ -0,0 +1,14 @@
+Value Filldown interface (.+) 
+Value errorcode (.+)
+Value timestamp (.+)
+Value description (.+)
+Value recommendation (.+)
+
+Start
+  ^Interface:\s+${interface} -> Clear
+  ^Error Code:\s+${errorcode}
+  ^Timestamp:\s+${timestamp}
+  ^Description:\s+${description}
+  ^Recommendation:\s+${recommendation} -> Record
+
```

Comment:
Need to match all possible lines, and add an error for any line that is not matched